docs/faq.md

```diff
@@ -92,13 +92,13 @@ As Semgrep evolves, queries similar to `foo("password")` become smarter. In the
 
 **3. Integrated: Semgrep understands git and other version-control systems**
 
-It’s easy to write a new Semgrep rule and have it only apply _going forward_. You can [ignore findings](../ignoring-files-folders-code) of course, but we have [built-in support for this with Semgrep CI](/semgrep-ci/overview/) and GitHub/GitLab/etc. integrations.
+It’s easy to write a new Semgrep rule and have it only apply _going forward_. You can [ignore findings](/ignoring-files-folders-code) of course, but we have [built-in support for this with Semgrep CI](/semgrep-ci/overview) and GitHub/GitLab/etc. integrations.
```

Comment:
I'm adding a task https://linear.app/semgrep/issue/MKT-896/audit-faq because this is not an up-to-date answer and the FAQ has been unevenly updated over time. (Some parts are older, some parts are newer...)